### PR TITLE
removed open and close and made the timing(closing in 1 min) colored …

### PR DESCRIPTION
**Pull request by pravalgautam**
…as red and green

Signed-off-by: Praval Gautam <pravalgautam@Pravals-MacBook-Air.local>
open close issue

### CODE REVIEW

File: iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift [hunk 1]
```diff
@@ -73,65 +72,65 @@ final class PlacePagePreviewViewController: UIViewController {
...
-
+  
   func updateDistance(_ distance: String) {
     self.distance = distance
     placePageDirectionView?.isHidden = false
     placePageDirectionView?.label.text = distance
     fullScreenDirectionView.updateDistance(distance)
   }
-
+
```
Comment (biodranik) [inline iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift:103]: Please don't reformat the existing code.

File: iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift [hunk 2]
```diff
@@ -23,15 +23,14 @@ final class PlacePagePreviewViewController: UIViewController {
   @IBOutlet var addressLabel: UILabel!
   @IBOutlet var addressContainerView: UIStackView!
   @IBOutlet var scheduleContainerView: UIStackView!
-
```
Comment (biodranik) [inline iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift:26]: Why did you remove it?

File: iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift [hunk 3]
```diff
@@ -165,9 +164,9 @@ final class PlacePagePreviewViewController: UIViewController {
         details = nil;
       }
       
-      setScheduleLabel(state: L("editor_time_open"),
+      setScheduleLabel(state: L(""),
```
Comment (biodranik) [inline iphone/Maps/UI/PlacePage/Components/PlacePagePreviewViewController.swift:167]: No. That is not a good fix. All unnecessary operations should be avoided.

**Review by biodranik — CHANGES_REQUESTED** [review]
Those dots don't look good.

**Comment by pravalgautam** [comment]
> Those dots don't look good.
